cyclestreets.app/src/main/AndroidManifest.xml

```diff
@@ -88,6 +88,10 @@
         <category android:name="android.intent.category.DEFAULT" />
       </intent-filter>
     </activity>
+    <activity
+        android:name="net.cyclestreets.views.CircularRouteActivity"
```

Comment:
Thanks for these comments.

What would you suggest otherwise, instead of the Activity?  

I originally thought to do an alert dialog, but the number of buttons (including cancel) exceeded the max.  (Bear in mind we still have to add in a button to select POI categories.)

Reply:
I'll ponder after seeing some screens. You may well be right, I just wanted to discuss